### PR TITLE
Remove ocagent and rename unisvc to otelsvc

### PR DESCRIPTION
**Pull request by flands**
Keep occollector for testing until migration to otelsvc

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,8 +16,8 @@ bin/
 
 # Binaries are copied (as needed) to the same location as their respective Dockerfile in
 # order to simplify docker build commands. Ignore these files if proper clean up fails.
-cmd/ocagent/ocagent_linux
 cmd/occollector/occollector_linux
+cmd/otelsvc/otelsvc_linux
```
Comment (pjanotti) [inline .gitignore]: Not for this PR but just crossed my mind that it is an opportunity to perhaps not use the $(GOOS) as suffix. Something like: by default we build otelsvc and separate by folder, eg.: ./bin/linux/otelsvc - a minor thing but to decide before first release.

File: README.md [hunk 2]
```diff
@@ -81,26 +86,26 @@ $ kubectl apply -f example/k8s.yaml
...
+described below. By default, the Collector has the `opencensus` receiver
+enabled, but no exporters configured.
 
 Build the Collector and start it:
 
 ```shell
-$ make collector
-$ ./bin/occollector_$($GOOS)
+$ make otelsvc
+$ ./bin/otelsvc_$($GOOS)
```
Comment (pjanotti) [inline README.md]: nit: not from your change but s/($GOOS)/$(go env GOOS)

File: Makefile [hunk 3]
```diff
@@ -81,17 +81,13 @@ vet:
...
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/ocagent_$(GOOS) $(BUILD_INFO) ./cmd/ocagent
-
 .PHONY: collector
 collector:
 	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/occollector_$(GOOS) $(BUILD_INFO) ./cmd/occollector
 
-.PHONY: unisvc
-unisvc:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/unisvc_$(GOOS) $(BUILD_INFO) ./cmd/unisvc
+.PHONY: otelsvc
```
Comment (SergeyKanzhelev) [inline Makefile]: let's call it otsvc

File: README.md [hunk 4]
```diff
@@ -50,18 +47,26 @@ is just configure and deploy the OpenTelemetry Service separately. The
 OpenTelemetry Service will then automatically collect traces and metrics and
 export to any backend of users' choice.
 
-Currently the OpenTelemetry Service consists of two components, [OpenTelemetry
-Agent](#opentelemetry-agent) and [OpenTelemetry Collector](#opentelemetry-collector).
+Currently the OpenTelemetry Service consists of a single binary and two
+deployment methods:
+
+1. [Agent](#opentelemetry-agent) running with the application or on the same host as the application
+2. [Collector](#opentelemetry-collector) running as a standalone application
```
Comment (pjanotti) [inline README.md]: suggestion, somthing along the lines: "typically as a layer of egress with larger buffers and higher resource usage".

File: .gitignore (hunk 1 above)
Comment (tigrannajaryan) [reply .gitignore]: I had the same thought. It looks odd to have _linux in the executable name.

File: Makefile (hunk 3 above)
Comment (tigrannajaryan) [reply Makefile]: @SergeyKanzhelev looks like the community is split on what prefix is best to use (see here in gitter https://gitter.im/open-telemetry/community?at=5d099d6f325b465cc3609206).

I think we keep this as is for now. Once we have an agreement on what prefix to use for OpenTelemetry it is easy to rename this.

**Comment by flands** [comment]
@songy23 this is part of the unification effort -- occollector will be removed as well once otelsvc is done. Then otelsvc can be deployed as an agent or collector depending on requirements.

**Comment by flands** [comment]
Merging as-is for now so not to block work. If name changes then will put up another PR.

**Pull request merged**